File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> lottos, vector<int> win_nums) {
    int count = 0;
    int zeroCount = 0;
    vector<int> answer;
    for (int i = 0; i < lottos.size(); i++) {
        if (lottos[i] == 0) {
            zeroCount++;
            continue;
        }
        for (int j = 0; j < win_nums.size(); j++) {
            if (lottos[i] == win_nums[j]) {
                count++;
            }
        }
    }
   
    if ((count + zeroCount) > 1) {
        answer.push_back(7 - (count + zeroCount));
    }
    else {
        answer.push_back(6);
    }
    if (count > 1) {
        answer.push_back(7 - count);
    }
    else {
        answer.push_back(6);
    }
    return answer;
}

File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.cs
using System;

public class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int count = 0;
    int zeroCount = 0;
    for(int i=0; i<lottos.Length; i++){
        if(lottos[i] == 0){
                zeroCount++;
                continue;
            }
        for(int j=0; j<win_nums.Length; j++){
            if(lottos[i] == win_nums[j]){
                count++;
            }
        }
    }
    int[] answer = new int[2];
    if((count + zeroCount) > 1){
        answer[0]=7-(count + zeroCount);
    }
    else{
        answer[0]=6;
    }
    if(count > 1){
        answer[1]= 7 - count;
    }
    else{
        answer[1]= 6;
    }
    return answer;
    }
}